PROTOCOL.md

```diff
@@ -82,7 +84,7 @@ The state of a table at a given version is called a _snapshot_ and is defined by
  - **Set of applications-specific transactions** that have been successfully committed to the table
 
 ## File Types
-A Delta table is stored within a directory and is composed of four different types of files.
+A Delta table is stored within a directory and is composed of five different types of files.
```

Comment:
I think we can just remove the number from here. its easy to forget to update this.